docs/docker-integration/cloudformation.rst

```diff
@@ -0,0 +1,167 @@
+.. _cloudformation:
+
+.. raw:: html
+
+    <style>
+        .toctree-wrapper { display:none; }
+    </style>
+
+=========================================================
+Installing Flocker with Swarm on AWS using CloudFormation
+=========================================================
+
+The steps in this guide enable you to quickly deploy a Flocker/Swarm cluster, as illustrated below, which will create four EC2 instances on your AWS account:
+
+.. raw:: html
+
+	<div style="width:80%; margin-left:auto; margin-right:auto; margin-bottom:2em;">
+
+.. figure:: ../images/cloudformation.png
+    :alt: A diagram illustrating a cluster of four AWS EC2 instances running Flocker with Docker Swarm.
+
+.. raw:: html
+
+	</div>
+
+Follow the steps below to create your cluster.
+Once it's up and running we'll guide you through a tutorial to deploy a sample app.
+
+.. source material for this image: https://drive.google.com/open?id=0ByymF9bLBknGeXlPX1pTdXVZOGM
+
+.. raw:: html
+
+	<div class="step-stages step-stages--3up">
+		<div class="step-stages__excerpt">
+			<h2 class="step-stages__heading">Step 1</h2>
```

Comment:
Can we please consider replacing ``Step 1`` with ``1``, similar to Mesosphere installation steps? Thanks!